cpp/client/ct.cc

```diff
@@ -753,7 +755,7 @@ static void DiagnoseCertChain() {
       continue;
     }
     LOG(INFO) << "SCT number " << i + 1 << ":\n" << sct.DebugString();
-    if (verifier != NULL) {
+    if (verifier.get()) {
```

Comment:
I think `unique_ptr` can be used directly in a conditional, no need to convert to pointer.